routes/users.go

```diff
@@ -34,6 +35,20 @@ type registerRequest struct {
 	DateOfBirth *time.Time `json:"dateOfBirth"`
 }
 
+type updateUserRequest struct {
+	PasswordHash    []byte
+	NameFirst       *string    `json:"firstName"`
+	NameLast        *string    `json:"lastName"`
+	Email           *string    `json:"email"`
```

Comment:
I don't think a user should be able to change their email since that's also a unique thing

Reply:
lets just not support it lol cause then updating profile may fail

Reply:
but if you really want to support it, you'll need to check if updating causes an error and if it does return a 409 conflict